disco_aws_automation/disco_storage.py

```diff
@@ -165,17 +165,15 @@ def configure_storage(self,
                           extra_space=None,
                           extra_disk=None,
                           iops=None,
-                          ephemeral_disk_count=0,
-                          map_snapshot=True):
-        '''Alter block device to destroy the volume on termination and add any extra space'''
+                          ephemeral_disk_count=0):
+        ''' Cofigures block device mapping for root partition, extra disk if it's used,
+        and the ephemeral discks. Note that EBS volume mapping is no longer being created
+        here. The logic for attaching EBS volume has been moved to /etc/asiaq/init/ebs-start.sh '''
```

Comment:
wait. so /etc/asiaq/init/eb-start.sh is only in asiaq_astro_config. Does this mean any config who did not follow up with asiaq_astro_config will be broken? This will be a breaking changes to down stream.